libraries/Microsoft.Bot.Builder/TurnContext.cs

```diff
@@ -164,17 +164,12 @@ public async Task<ResourceResponse> SendActivityAsync(string textReplyToSend, st
                 throw new ArgumentNullException(nameof(textReplyToSend));
             }
 
-            var activityToSend = new Activity(ActivityTypes.Message) { Text = textReplyToSend };
-
-            if (!string.IsNullOrEmpty(speak))
+            var activityToSend = new Activity(ActivityTypes.Message)
             {
-                activityToSend.Speak = speak;
-            }
-
-            if (!string.IsNullOrEmpty(inputHint))
-            {
-                activityToSend.InputHint = inputHint;
-            }
+                Text = textReplyToSend,
+                Speak = !string.IsNullOrEmpty(speak) ? speak : textReplyToSend,
+                InputHint = !string.IsNullOrEmpty(inputHint) ? inputHint : InputHints.ExpectingInput,
```

Comment:
>ExpectingInput [](start = 86, length = 14)

I don't think we can make this change.  If it is null we should pass null and have the channel interpret null to be whatever default makes sense to it in the context of the channel.